src/sqlfluff/dialects/dialect_mysql.py

```diff
@@ -314,6 +330,142 @@ class CreateTableStatementSegment(ansi.CreateTableStatementSegment):
     )
 
 
+class CreateUserStatementSegment(ansi.CreateUserStatementSegment):
+    """`CREATE USER` statement.
+
+    https://dev.mysql.com/doc/refman/8.0/en/create-user.html
+    """
+
+    _string_literal = OneOf(
+        Ref("QuotedLiteralSegment"), Ref("DoubleQuotedLiteralSegment")
+    )
```

Comment:
Should this be globally available to all of MySQL?

In fact should it be added to `QuotedLiteralSegment`?

Reply:
Did this: https://github.com/sqlfluff/sqlfluff/pull/3289/commits/dbbd96802905cd401d2bf011dcbf0dcfcbc8d761

It's possible it could be iterated on even more.